src/sources/GeoJSON.js

```diff
@@ -423,7 +432,7 @@ export default class GeoJSON extends Base {
             total += (pt2[1] - pt1[1]) * (pt2[0] + pt1[0]);
             pt1 = pt2;
         }
-        return total >= 0;
+        return total >= 0; // CW
```

Comment:
Instead of a comment, create a private function:

```js
function _isClockWise (value) {
    return value >= 0;
}
```

Reply:
I don't think this is needed, because this function does not check ClockWise, only if the value is positive because there is no context. Also, it adds an extra function call for each point and it could add a bit of extra computing time.

I could improve the comment instead.